migrations/202407171220_add_response_state_to_request_events.go

```diff
@@ -0,0 +1,33 @@
+package migrations
+
+import (
+	"github.com/go-gormigrate/gormigrate/v2"
+	"gorm.io/gorm"
+)
+
+// Add response_received_at column to request_events table
+var _202407171220_add_response_received_at_to_request_events = &gormigrate.Migration{
+	ID: "202407171220_add_response_received_at_to_request_events",
+	Migrate: func(tx *gorm.DB) error {
+		if err := tx.Exec("ALTER TABLE request_events ADD COLUMN response_received_at TIMESTAMP DEFAULT '0001-01-01 00:00:00+00'").Error; err != nil {
```

Comment:
I don't think a timestamp should have such a default, why did you add this? 

it can be nil. 

nil means no response received
and a timestamp indicates that and when a response is received

Reply:
I thought it would be easy to handle it in go if we wish to do something with these values later, but yeah it's not that much of a hassle and it's actually better while querying in metabase 👍

Reply:
>  thought it would be easy to handle it in go if we wish to do something with these values later, 

what does that mean? 
this is in the DB and you have to think about correct DB values.